src/index.js

```diff
@@ -1,9 +1,15 @@
 'use strict';
 
-export function annotate (...services) {
+var slice = [].slice;
+
+function annotate () {
+  var services = slice.call(arguments);
   return function decorate (target, key, descriptor) {
     if (descriptor) target = descriptor.value;
     target.$inject = services;
     return descriptor;
   }
 }
+
+annotate.annotate = annotate;
```

Comment:
Last thing, can you just export the function? Happy to do a major release and get it right.